ui/src/spa_index.jsx

```diff
@@ -0,0 +1,509 @@
+      fltr = [];
+    }
+    setWindow(searchParams.get('window') || '6d');
+    setAggregateBy(searchParams.get('agg') || 'namespace');
+    setAccumulate((searchParams.get('acc') === 'true') || false);
+    setIdle(searchParams.get('idle') || 'separate');
+    setTitle(searchParams.get('title') || 'Last 7 days by namespace daily');
+    setShareSplit(searchParams.get('split') || 'weighted');
+    setContext(ctx);
+    setDetails(deets);
+    setFilters(fltr);
+  }, [routerLocation]);
+
+  async function initialize() {
+    setInit(true)
+  }
+
+  async function fetchData() {
+    setLoading(true)
+    setErrors([])
+
+    try {
+      let queryFilters = []
+      if (context.length > 0) {
+        forEach(context, (contextFilter) => {
+          queryFilters.push(contextFilter)
+        })
+      }
+      forEach(filters, (filter) => {
+        queryFilters.push(filter)
+      })
+
+      const resp = await AllocationService.fetchAllocation(window, aggregateBy, {
+        accumulate: accumulate,
+        filters: queryFilters,
+      })
+      if (resp.data && resp.data.length > 0) {
+        const allocationRange = resp.data
+        for (const i in allocationRange) {
+          // update cluster aggregations to use clusterName/clusterId names
+          if (aggregateBy == "cluster") {
+            for (const k in allocationRange[i]) {
+              allocationRange[i][k].name = 'cluster-one';
+            }
+          }
+          allocationRange[i] = sortBy(allocationRange[i], a => a.totalCost)
+        }
+        setAllocationData(allocationRange)
+      } else {
+        if (resp.message && resp.message.indexOf('boundary error') >= 0) {
+          let match = resp.message.match(/(ETL is \d+\.\d+% complete)/)
+          let secondary = 'Try again after ETL build is complete'
+          if (match.length > 0) {
+            secondary = `${match[1]}. ${secondary}`
+          }
+          setErrors([{
+            primary: "Data unavailable while ETL is building",
+            secondary: secondary,
+          }])
+        }
+        setAllocationData([])
+      }
+    } catch (err) {
+      if (err.message.indexOf("404") === 0) {
+        setErrors([{
+          primary: "Failed to load report data",
+          secondary: "Please update Kubecost to the latest version, then contact support if problems persist."
+        }])
+      } else {
+        let secondary = "Please contact Kubecost support with a bug report if problems persist."
+        if (err.message.length > 0) {
+          secondary = err.message
+        }
+        setErrors([{
+          primary: "Failed to load report data",
+          secondary: secondary,
+        }])
+      }
+      setAllocationData([])
+    }
+
+    setLoading(false)
+    setFetch(false)
+  }
+  return (
+    <Page active="reports.html">
+      <Header breadcrumbs={[
+        { 'name': 'Reports', 'href': 'new_index.html#/reports' },
+        { 'name': title, 'href': `new_index.html#/reports` },
+      ]}>
```

Comment:
I think we can drop the breadcrumb. What do you think?

Reply:
If we aren't drilldown-ing then absolutely. So yes.